source/features/sticky-csv-header.css

```diff
@@ -0,0 +1,33 @@
+.markdown-body .csv-data {
+	max-height: 100vh;
```

Comment:
On second thought… nested scroll areas aren't my favorite and this only works well if you perfectly align the scroll area to the viewport. It shouldn't be necessary

Reply:
It is necessary, since that element is already scrolling horizontally.